packages/components/date-picker/__tests__/date-picker.spec.ts

```diff
@@ -373,6 +373,46 @@ describe('DatePicker', () => {
       await nextTick()
       expect(wrapper.findComponent(Input).vm.modelValue).toBe('2021-05-31')
     })
+
+    it('with "timestamp"', async () => {
+      const format = 'YYYY-MM-DD'
+      const valueFormat = 'timestamp'
+      const value = Date.now()
+      const wrapper = _mount(
+        `
+        <el-date-picker
+          ref="compo"
+          v-model="value"
+          type="date"
+          format="${format}"
+          value-format="${valueFormat}" />
+        <button @click="changeValue">click</button>
+      `,
+        () => {
+          return {
+            value,
+          }
+        },
+        {
+          methods: {
+            changeValue() {
+              this.value = +new Date('2021/05/31')
```

Comment:
Let's move this `'2021/05/31'` to a variable so that we can assert it in the upcoming assertions.